server/src/settings/Settings.ts

```diff
@@ -0,0 +1,62 @@
+/** Describe the YML relevant settings part as defined in `package.json`. */
+export interface IYseopmlSettings {
+    yseopml: IYseopmlServerSettings;
+}
+
+/** Describe the _server_ relevant settings part as defined in `package.json`. */
+export interface IYseopmlServerSettings {
+    activateParsingProblemsReporting: boolean;
+    parseAllProjectFilesAtStartup: boolean;
+    pathToPredefinedObjectsXml: string;
+    ymlParsingIssueSeverityLevel: string;
+    kaoFiles: string[];
+    documentFormat: IDocumentFormatSettings;
+}
+
+/**
+ * Possible attributes that can be sent as document format settings as defined in `package.json`.
+ */
+export interface IDocumentFormatSettings {
+    // 'yes' and 'no' acts like a new type. Impossible to use const.
+    /** `yes` to enable the document format feature. `no` otherwise. */
+    enableDocumentFormat: 'yes' | 'no';
+    /** `true` to add semicolon when possible. `false` otherwise. */
+    semicolonWhenPossible: boolean;
+    /**
+     * `true` to set a one space distance between some keywords and parenthesis. \
+     * `false` to have this distance set to zero.
+     */
+    spaceBetweenKeywordAndParenthesis: boolean;
+    // More options to come.
```

Comment:
At least, I'll have to add an option for the feature that set one space around `&&` `||` and such elements.

Reply:
> `&&` `||` and such elements.

You mean “operators”? :trollface:
But seriously, I think you had grammar rules dedicated to conditions and boolean operators, so I,m genuinely wondering if you’ll be able to hadle all operators at once. Oh and some things like `.` or whatever may technically count as operators, in some contexts, I think :thinking: but obviously they do not follow the same spacing rules.

Reply:
“operators”, indeed.
In fact, I already handle − I think − all of those operators (`.` is not an operator in our case − or at least not defined as an operator).
I just don't have an option to remove this feature.

Reply:
LOLed in the open space.